src/components/Blogs/index.scss

```diff
@@ -0,0 +1,56 @@
+@import "../../style/variables";
+
+.blogsMainContainer {
+  text-align: center;
+  font-family: $mainFontFamily;
+
+  .container {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+  }
+
+  .col {
```

Comment:
i didn't this col in jsx file. where does it come from?

Reply:
these from bootstrap, I will comments if that's allowed, will ask Ammar about it.

Reply:
but you are not using it anywhere. it's not even imported in index.jsx file

Reply:
it comes with the package I guess, I am not sure. however when I inspected it and tried to style it worked.